pkg/awsclient/client.go

```diff
@@ -46,6 +48,16 @@ const (
 	awsCredsSecretAccessKey = "aws_secret_access_key"
 )
 
+var (
+	metricAWSAPICalls = prometheus.NewCounterVec(
```

Comment:
@dgoodwin should this be registered somewhere?

Reply:
LOL yes thank you. My bad.

Reply:
Updated.